src/converse-chatview.js

```diff
@@ -543,6 +555,8 @@ converse.plugins.add('converse-chatview', {
                     'show_spoiler_button': _converse.visible_toolbar_buttons.spoiler,
                     'tooltip_insert_smiley': __('Insert emojis'),
                     'tooltip_start_call': __('Start a call'),
+                    'label_file_download': __('Start Download-Dialog'),
+                    'show_download_button': _converse.visible_toolbar_buttons.download,
```

Comment:
You can override `getToolbarOptions` in src/converse-attachement-download.js to add this key.